pkg/policy/policy.go

```diff
@@ -73,23 +73,31 @@ func NewMetadata(values map[string]interface{}) (Metadata, error) {
 	}, nil
 }
 
-// Result describes result of evaluating a policy.
+// Result describes result of evaluating a Rego policy that defines `deny` or
+// `warn` rules.
 type Result struct {
-	Message string
+	// Metadata describes Rego policy metadata.
+	Metadata Metadata
+
+	// Success represents the status of evaluating Rego policy.
+	Success bool
+
+	// Messages deny or warning messages.
+	Messages []string
 }
 
-// NewResult constructs new Result based on raw values.
-func NewResult(values map[string]interface{}) (Result, error) {
+type Results []Result
+
+// NewMessage constructs new message string based on raw values.
+func NewMessage(values map[string]interface{}) (string, error) {
 	if values == nil {
-		return Result{}, errors.New("values must not be nil")
+		return "", errors.New("values must not be nil")
 	}
 	message, err := requiredStringValue(values, "msg")
```

Comment:
maybe make "msg" a constant with description comment just to make it clear for new readers

Reply:
Okey, I used constant for  the msg variable.